lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -475,6 +502,8 @@ def create_network(self):
                 params['scope'] = self.parameters.scope
             if self.parameters.attachable is not None:
                 params['attachable'] = self.parameters.attachable
+            if self.parameters.labels is not None:
```

Comment:
It will never be `None`, since you set the default value to `{}` below. Changing this to simply `if self.parameters.labels:` fixes this, or changing the default value to `None`.